serverless.yml

```diff
@@ -1,34 +1,38 @@
 service: lucene-serverless
-variablesResolutionMode: 20210326
+variablesResolutionMode: 20210219
 
+# TODO: replace all variables below
 custom:
   name: ${sls:stage}-${self:service}
   region: ${opt:region, self:provider.region}
-  vpcId: vpc-8aa25ee3 # TODO: replace with your VPC settings region
-  subnetId1: subnet-efd7e8a5 # TODO: replace with your VPC settings region
-  subnetId2: subnet-7a4b5702 # TODO: replace with your VPC settings region
-  subnetId3: subnet-486d9f21 # TODO: replace with your VPC settings region
+  vpcId: vpc-00690f6e
+  subnetId1: subnet-039f496e
+  subnetId2: subnet-0c0af8ea
+  region: us-east-1
```

Comment:
Serverless validate complains about this being a duplicate declaration

Reply:
Removed the duplicate! Copied that over wrong from my project

>  I was also thinking if the binary should be committed into this project so that it's easy to deploy without setting up the whole build environment.

Ah the change set excludes `target/function.zip` per `.gitignore`, should I add it back?

Reply:
Please rebuild with the latest code and push the new `function.zip`. Thanks!